src/yum-plugins/tracer_upload.py

```diff
@@ -1,36 +1,31 @@
-import sys
 import time
 
-try:
-  from tracer.query import Query
-except ImportError:
-  sys.exit('Error Importing tracer! Is tracer installed?')
+from katello.tracer import Query, upload_tracer_profile
 
-from yum.plugins import PluginYumExit, TYPE_CORE, TYPE_INTERACTIVE
-from katello.tracer import upload_tracer_profile
+from yum.plugins import TYPE_CORE, TYPE_INTERACTIVE
 
 requires_api_version = '2.3'
 plugin_type = (TYPE_CORE, TYPE_INTERACTIVE)
 
+
 def query_apps(conduit):
```

Comment:
@sean797 the conduit is used in this method, via the yum plugin. I just verified that we are still hitting this code path